examples/overlay-warnings/README.md

```diff
@@ -0,0 +1,11 @@
+# Overlay
+
+```shell
+node ../../bin/webpack-dev-server.js --open
+```
+
+## What should happen
+
+The script should open the browser and show a heading with "Example: overlay with warnings".
+
+In `app.js`, make a syntax error. The page should now refresh and show a full screen error overlay that shows the syntax error.
```

Comment:
Jup, in webpack 1 that triggered a warning but in webpack 2 not anymore :'). Looking at webpack's source code on `warnings.push`, I found that it should trigger a warning when incorrectly cased modules are used. So perhaps add a file `caseSensitive.js` and import it with `casesensitive.js`?